chase/frontend/components/dashboard/speakerslist.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import WidgetTemplate from "@components/dashboard/widgettemplate";
+import SpeakerBlock from "@components/dashboard/speakerslist/speakerblock";
+import CommentBlock from "@/components/dashboard/speakerslist/commentblock";
+import QueueBlock from "@/components/dashboard/speakerslist/queueblock";
+
+export default function SpeakersListWidget() {
+  // Demo Data
+  // TODO remove
+  const myCountry = "jam";
+
+  const currentSpeaker = {
+    countryCode: "uno",
+    countryName: "Generalsekretär",
+    time: "2:35",
+  };
+
+  const speakersList = ["gbr", "che", "yem", "fra", "jam"];
+
+  const currentComment = {
+    countryCode: "cze",
+    countryName: "Republik Tschechien",
+    time: "0:23",
+  };
+
+  // const currentComment = null;  // If there is no data for current comment, the commentlist block will not be rendered
+
+  const commentList = ["deu", "jam", "usa", "yem"];
+
+  return (
+    <>
+      <WidgetTemplate cardTitle="Redeliste">
+        <div className="flex-1 flex flex-col gap-3">
+          <SpeakerBlock
+            countryCode={currentSpeaker.countryCode}
+            countryName={currentSpeaker.countryName}
+            time={currentSpeaker.time}
```

Comment:
I'd suggest increasing the font size of the timer a little to give it more visual presence

Reply:
After all its just the preview. If you want a full grasp on the Timing you should navigate to the speakers list page.